### PR TITLE
VMware-Hosted-Private-Cloud-VCD - Adding-changes-to-one-branch-for-review

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Comment by BrenKt** [comment]
- Content FR reviewed
- GIFs to resize
- EN translation to do after FR validation

**Comment by BrenKt** [comment]
⚠️ Work in progress, review to redo after validation ⚠️

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md [hunk 1]
```diff
@@ -6,84 +6,219 @@ updated: 2024-04-16
...
 
 ## Prérequis
 
 >[!primary]
 > Si vous ne savez comment vous connecter au portail web de votre organisation, consultez d'abord [ce guide](/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-logging).
 
-- Avoir un compte VMware Cloud Director
+- Une connaissance pratique des réseaux Linux, vSphere, Windows et IP est nécessaire pour configurer et gérer vCD
+- Avoir un compte VMware Cloud Director administrateur
+- Avoir une infrastructure VMware Cloud Foundation (VCF)
```
Comment (pg-ovh) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md]: Hello @franckcosta est-ce que j'enlève cette ligne (20) en réf à "VCF" pour l'instant ?

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md [hunk 2]
```diff
@@ -6,84 +6,219 @@ updated: 2024-04-16
...
+VMware vCloud Director (VCD) permet d'avoir un service reuni au sein d'une même organisation managé par OVHcloud.
 
 >[!primary]
-> Une fois connecté à votre interface web, vous serez accueilli par un tableau de bord affichant vos **vDC**, ainsi qu'un résumé détaillé de votre utilisation des ressources (10). En haut de l'écran, vous trouverez également une barre de navigation regroupant les différentes options de paramétrage disponibles pour vCloud Director (VCD).
+> Une fois connecté à votre interface web, vous serez accueillis par un tableau de bord affichant vos **vDC**, ainsi qu'un résumé détaillé de votre utilisation des ressources (10). En haut de l'écran, vous trouverez également une barre de navigation regroupant les différentes options de paramétrage disponibles pour vCloud Director (VCD).
 
 ![Dashboard Overview](images/vcd-dashboard-overview.png){.thumbnail}
 
-Les différentes sections disponibles :
+<details>
```
Comment (OVH-Team-Guides) [inline pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md]: @pg-ovh , I suggest you remove all the `<details>` formats from this guide as they are not necessary.

If they were to be kept, they would need fixes as there are too many `<summary>` tags and all included content has to be formatted in HTML too, as in https://help.ovhcloud.com/csm/fr-vmware-nsx-interface-user-access?id=kb_article_view&sysparm_article=KB0062999 (see formats in https://github.com/ovh/docs/blob/develop/pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/nsx_add_user_rights/guide.fr-fr.md?plain=1)

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md (hunk 1 above)
Comment (franckcosta) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md]: oui

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md (hunk 2 above)
Comment (pg-ovh) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md]: @OVH-Team-Guides also from the en-gb guide ?

**Comment by pg-ovh** [comment]
Hello @OVH-Team-Guides I have removed the changes made without any formatting, let me know if you need more help on this PR. Edition done.

File: pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md (hunk 2 above)
Comment (OVH-Team-Guides) [reply pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vcd-getting-started/guide.fr-fr.md]: Yes please

**Comment by pg-ovh** [comment]
Hello @OVH-Team-Guides, I seem to have broken the folder structure by trying to merge my local conflicts. I have informed @tcpdumpfbacke in pm. Let me know if you need my local changes from this commit : https://github.com/ovh/docs/compare/19e908c87953faeed91e815b516a5d398e04510a..bb9d5be306755023bd80156c05d526a4a071b517
Sorry for the inconvenience.

**Comment by OVH-Team-Guides** [comment]
Replaced by #6349

**Pull request closed without merging**